src/proto/admin_rpc.proto

```diff
@@ -5,16 +5,8 @@ import "username_proof.proto";
 
 message Empty {}
 
-message TerminateRequest {
-  bool destroy_database = 1;
-}
-
-message TerminateResponse {
-}
-
 service AdminService {
-  rpc Terminate(TerminateRequest) returns (TerminateResponse);
-  rpc SubmitOnChainEvent(OnChainEvent) returns (OnChainEvent);
-  rpc SubmitUserNameProof(UserNameProof) returns (UserNameProof);
+//  rpc SubmitOnChainEvent(OnChainEvent) returns (OnChainEvent);
+//  rpc SubmitUserNameProof(UserNameProof) returns (UserNameProof);
```

Comment:
These were meant for debugging and should not be allowed on mainnet. Remove for now, we can add back in a `DebugService` later if required (which is not mounted if the configured network is mainnet).